playbook/app/pb_kits/playbook/pb_title/docs/_title_responsive.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Title from '../_title'
+
+const TitleResponsive = (props) => {
+  return (
+    <>
+      <Title
+          size={{xs: "3", sm: "2", md: "1"}}
+          text="Responsive Title"
+          {...props}
+      />
+    </>
+  )
+}
+
+export default TitleResponsive
```

Comment:
looks good @carloslimasd ! Great work on this one! I think we should definitely add in some copy for the doc examples though, we can ask Colin to provide some.